spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-appactive/src/main/java/com/alibaba/cloud/appactive/consumer/ReactiveRequestStrategyBeanPostProcessor.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2013-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.cloud.appactive.consumer;
+
+import javax.validation.constraints.NotNull;
+
+import com.alibaba.cloud.appactive.constant.AppactiveConstants;
+import io.appactive.java.api.base.AppContextClient;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.web.reactive.function.client.ClientRequest;
+import org.springframework.web.reactive.function.client.WebClient;
+
+/**
+ * @author yuluo
+ */
+public class ReactiveRequestStrategyBeanPostProcessor implements BeanPostProcessor {
```

Comment:
How to reload the class? Please check how we initialize the FeignDecoderPostProcessor. BTW, if you can add relevant web client example, it be better. Otherwise, it's hard to check whether the logic is alright.

Reply:
ok

Reply:
> How to reload the class? Please check how we initialize the FeignDecoderPostProcessor. BTW, if you can add relevant web client example, it be better. Otherwise, it's hard to check whether the logic is alright.

For WebClient-example, Please See #3140